### PR TITLE
ClusterSSH: update to 4.13.2.01.

### PR DESCRIPTION
**Pull request by Chocimier**

### CODE REVIEW

File: srcpkgs/ClusterSSH/template [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
 maintainer="Enguerrand de Rochefort <voidlinux@rochefort.de>"
-homepage="http://clusterssh.sourceforge.net"
-license="GPL-1, Artistic"
-distfiles="${SOURCEFORGE_SITE}/clusterssh/App-${pkgname}-${version}.tar.gz"
-checksum=eb0df729340054a787429cfcefac79d28cf9013553d01d079ffd8e82fbf35681
+license="Artistic-1.0-Perl, GPL-1.0-or-later"
+homepage="https://github.com/duncs/clusterssh"
+distfiles="https://github.com/duncs/clusterssh/archive/v${_distver}.tar.gz"
+checksum=be6da42d448d925485d24c060f93215980e4e603476d61e18e79e219facea65a
+nocross=yes
```
Comment (jnbr) [inline srcpkgs/ClusterSSH/template]: why? can't this be fixed?

Comment (Chocimier) [reply srcpkgs/ClusterSSH/template]: It's noarch, nocross is really problem?

Comment (jnbr) [reply srcpkgs/ClusterSSH/template]: yes, with our current build system, noarch packages need to be built for every architecture to be added to the repodata

**Comment by Chocimier** [comment]
Error message suggests problem with build style.

**Comment by Hoshpak** [comment]
Adding perl to makedepends makes the build succeed. Since the perl binary from the target architecture is not executed during the build, I guess this is alright. It just needs the perl scripts in place to build.

**Comment by Chocimier** [comment]
As it is used by build style, let's fix it there: #7331

**Comment by Chocimier** [comment]
Updated, as build style fix is merged.

**Pull request merged**